addons/website_sale_slides/models/sale_order.py

```diff
@@ -29,6 +29,6 @@ def _action_confirm(self):
                         channels_per_so[so_line.order_id] = channels_per_so[so_line.order_id] | related_channel
 
         for sale_order, channels in channels_per_so.items():
-            channels._action_add_members(sale_order.partner_id)
+            channels.sudo()._action_add_members(sale_order.partner_id)
```

Comment:
Currently, When the user has the right in sales to edit his own documents, or all documents, but has no rights on the website he is able to validate the payment for a course, put this will not put the user who bought the course in the
attendees of the course, as he could not modify those.
so I added the sudo to make sure that whenever a sale is confirmed, the course will be given to the payer.

Reply:
Does this issue only occur in your branch?

If not, it should be extracted to another FIX branch based on the related stable branch (saas-12.3 or 13.0 depending on when the issue appeared first).
You will also have to create a separate task in our project, explain the use case, and make it tested separately.

It's a good thing to fix the issues you encounter, but we have to make sure they're fixed for older branches too.
So the clients that are in production on these older branches will receive the fix as well.

If that's unclear, don't hesitate to ask.

Reply:
No it's clear, i'll have a look at saas-12.3 and 13.0 :p

Reply:
Bon bein, I don't have this issue anymore neither on my branch neither on the other ones so I removed the sudo